root/report/VotingReportsIndex.js

```diff
@@ -0,0 +1,363 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {withCatalystContext} from '../context';
+import Layout from '../layout';
+
+type VotingReportRowProps = {
+  +noOwnEditsUrl?: string,
+  +noVoteYetUrl?: string,
+  +reportName: string,
+  +reportUrl: string,
+  +subscribedArtistsUrl?: string,
+  +subscribedNoVoteYetUrl?: string,
+};
+
+const VotingReportRow = ({
+  noOwnEditsUrl,
+  noVoteYetUrl,
+  reportName,
+  reportUrl,
+  subscribedArtistsUrl,
+  subscribedNoVoteYetUrl,
+}: VotingReportRowProps) => (
+  <li>
+    <a href={reportUrl}>
+      {reportName}
+    </a>
+    {noOwnEditsUrl ? (
+      <ul>
+        <li>
+          <a href={noOwnEditsUrl}>
+            {l('…by editors other than me')}
+          </a>
+          {noVoteYetUrl ? (
+            <ul>
+              <li>
+                <a href={noVoteYetUrl}>
+                  {l('…that I haven’t voted on yet')}
+                </a>
+              </li>
+            </ul>
```

Comment:
This `ul` should be merged with the one below, since they're at the same level and part of the same semantic list. It wouldn't look visually different, but def. creates a confusing page structure for screen readers in particular.

There appear to be similar cases below.